.github/workflows/tests.yml

```diff
@@ -74,7 +77,7 @@ jobs:
 
   # stolen from https://raw.githubusercontent.com/exercism/github-actions/main/.github/workflows/shellcheck.yml
   shellcheck:
-    name: shellcheck internal tooling
+    name: shellcheck internal tooling lint
```

Comment:
Renamed this for consistency. I should also rename our `markdown_lint.sh` to `lint_markdown.sh`. It seems most of our internal tooling follows a `category-subcategory` naming scheme.